### DISCUSSION TITLE
posts/reconstruct-my-blog-with-hugo-solidstart/

### DISCUSSION THREAD

**Discussion by giscus** [WincerChan/blog-comments#4] [Announcements]
# posts/reconstruct-my-blog-with-hugo-solidstart/

在去年年底时，我 重新设计 了一版博客的主题，不过此版主题主要是针对 UI 样式方面的调整，底层的架构没有变化：静态内容还是 Hugo 生成，动态内容如：搜索、消遣页面，暗色模式、评论等功能则是由 Svelte 提供支持。这个技术栈使用起来其实挺搭的：Svelte 打包后的结果很小，生成的 JavaScript 文件直接在 Hugo 模板中用一个 script 标签引入就行了。
那么，是什么原因让我仅仅时隔半年的时间就再次重构博客呢？让我先卖一个关子，先说一下 Hugo 的一些问题。

https://blog.itswincer.com/posts/reconstruct-my-blog-with-hugo-solidstart/

**Comment by yzyyz1387** [comment]
留下了没有技术了泪水（悲）

**Reply by WincerChan** [reply]
哈哈哈，随便折腾折腾
